diff --git a/node_modules/electron-webrtc/src/RTCDataChannel.js b/node_modules/electron-webrtc/src/RTCDataChannel.js
index 825da40..14eab52 100644
--- a/node_modules/electron-webrtc/src/RTCDataChannel.js
+++ b/node_modules/electron-webrtc/src/RTCDataChannel.js
@@ -211,6 +211,12 @@ module.exports = function (daemon, wrtc) {
       return this._binaryType
     }
     set binaryType (value) {
+      // This is a patch to fix a bug in electron-webrtc that causes a crash
+      // when the binaryType is set to 'blob'. We convert it to 'arraybuffer'
+      // which is the only other supported type and is handled correctly.
+      if (value === 'blob') {
+        value = 'arraybuffer'
+      }
       this._binaryType = value
       this._setProp('binaryType', value)
     }
